Material.Styles/DialogHost.axaml

```diff
@@ -1,17 +0,0 @@
-<Styles xmlns="https://github.com/avaloniaui"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:dialogHost="clr-namespace:DialogHost;assembly=DialogHost.Avalonia"
-        xmlns:styles="clr-namespace:Material.Styles"
-        xmlns:assists="clr-namespace:Material.Styles.Assists">
-    <Style Selector="dialogHost|DialogHost">
```

Comment:
Might we keep this fragment and paste it on wiki page or create another package for it?

Reply:
No, we don't need it. It works fine out of the box with Material. Moreover, I want to create support for Material on the DialogHost.Avalonia side

Reply:
ok then its good to go